rasa/core/brokers/pika.py

```diff
@@ -389,10 +441,7 @@ def publish(
             retries -= 1
             time.sleep(retry_delay_in_seconds)
 
-        logger.error(
-            "Failed to publish Pika event to queue '{}' on host "
-            "'{}':\n{}".format(self.queue, self.host, body)
-        )
+        logger.error(f"Failed to publish Pika event on host '{self.host}':\n{body}")
```

Comment:
Do we not have the information about which queue failed to publish?

Reply:
not without completely restructuring the error handling. I think it would be misleading to say "failed to publish to queue X" as that will most likely just always be the first queue. the error is likely due to the connection as a whole, not a specific queue